testfiles/AbsorptionChiller.idf

```diff
@@ -108,7 +108,9 @@
   Timestep,6;
 
   ZoneCapacitanceMultiplier:ResearchSpecial,
-    1.0,                     !- Temperature Capacity Multiplier
+    ,      					 !- Name
```

Comment:
This will need a transition rule.  Will you add that, or do you need me or @Myoldmopar to do that?